kgcl/diff/change_detection.py

```diff
@@ -68,9 +66,10 @@ def detect_renamings(added, deleted):
             moved_to = added_labels[subject]
 
             if len(moved_to) > 1 or len(moved_from) > 1:
-                nonDeterministic.append((set(moved_from), set(moved_to)))
+                non_deterministic.append((set(moved_from), set(moved_to)))
 
             shared = min(len(moved_to), len(moved_from))
+            # TODO: What does x do here?
```

Comment:
@cmungall , would you know the answer to this?

Reply:
can this just be an `_` ?

Reply:
I don't know, maybe @ckindermann can shed light on this?

I would say though to go ahead and merge what you have

Reply:
Yes, this can be an `_`.